<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kappaleWidget</class>
 <widget class="QWidget" name="kappaleWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>170</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1000000</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Kehonkuva</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QProgressBar" name="jalatBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QProgressBar" name="kadetBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QProgressBar" name="twistBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QProgressBar" name="selkaBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QProgressBar" name="taipuvaBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QProgressBar" name="vatsaBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QProgressBar" name="ylosBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QProgressBar" name="rintakehaBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QProgressBar" name="materiaaliBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QProgressBar" name="paaBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QProgressBar" name="hartiatBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QProgressBar" name="arkkityyppiBar">
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Muoto</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QProgressBar" name="pulleaBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="eroavaisuus">
       <property name="text">
        <string>eroavaisuus</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QProgressBar" name="variBar">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="GLWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
